modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDumper.java

```diff
@@ -501,11 +537,11 @@ private StoreWriter getStoreWriter(
      *     geometry type suffix
      * @return
      */
-    protected String getShapeName(SimpleFeatureType schema, String geometryType) {
+    protected String getShapeName(String fileName, String geometryType) {
```

Comment:
Ah, I did miss this change in the review, and it's breaking the GeoServer build in the WFS module....

Reply:
Also, GeoServer really needs a type name, not a random filename

Reply:
Will wait a bit to see if by any chance you can respond quickly, and revert the merge by end of day otherwise.